cloudbridge/cloud/providers/gce/resources.py

```diff
@@ -1384,7 +1384,7 @@ def region(self):
 
     @property
     def zone(self):
-        raise NotImplementedError('To be implemented')
+        raise NotImplementedError('GCP subnetworks are regional resources')
```

Comment:
The zone is an optional parameter mainly for informational purposes so it's probably better to return None here.

Reply:
Done: fixed in #74.